flow/designs/sky130hd/coyote_tc/pad.tcl

```diff
@@ -0,0 +1,250 @@
+place_pad -master sky130_ef_io__vddio_hvc_pad -row IO_WEST -location 3260.0 {u_v18_17}
+place_pad -row IO_WEST -location 3180.0 {u_fsb_node_data_i.u_io\[5\].u_in}
+place_pad -row IO_WEST -location 3100.0 {u_fsb_node_data_i.u_io\[6\].u_in}
+place_pad -row IO_WEST -location 3020.0 {u_fsb_node_data_i.u_io\[7\].u_in}
+place_pad -row IO_WEST -location 2940.0 {u_fsb_node_ready_o.u_io}
+place_pad -row IO_WEST -location 2860.0 {u_fsb_node_v_o.u_io}
+place_pad -row IO_WEST -location 2780.0 {u_fsb_node_data_o_0.u_io}
+place_pad -master sky130_ef_io__vssd_hvc_pad -row IO_WEST -location 2700.0 {u_vss_7}
+place_pad -master sky130_ef_io__vccd_hvc_pad -row IO_WEST -location 2620.0 {u_vdd_7}
+place_pad -row IO_WEST -location 2540.0 {u_fsb_node_data_o_1.u_io}
+place_pad -row IO_WEST -location 2460.0 {u_fsb_node_data_o_2.u_io}
+place_pad -master sky130_ef_io__vssio_hvc_pad -row IO_WEST -location 2380.0 {u_vzz_18}
+place_pad -master sky130_ef_io__vddio_hvc_pad -row IO_WEST -location 2300.0 {u_v18_18}
+place_pad -row IO_WEST -location 2220.0 {u_fsb_node_data_o_3.u_io}
+place_pad -row IO_WEST -location 2140.0 {u_fsb_node_data_o_4.u_io}
+place_pad -row IO_WEST -location 2060.0 {u_fsb_node_data_o_5.u_io}
+place_pad -row IO_WEST -location 1980.0 {u_fsb_node_data_o_6.u_io}
+place_pad -row IO_WEST -location 1900.0 {u_fsb_node_data_o_7.u_io}
+place_pad -row IO_WEST -location 1820.0 {u_fsb_node_yumi_i.u_in}
+place_pad -row IO_WEST -location 1740.0 {u_rocc_ctrl_i_busy_.u_in}
+place_pad -row IO_WEST -location 1660.0 {u_rocc_ctrl_i_interrupt_.u_in}
+place_pad -master sky130_ef_io__vssio_hvc_pad -row IO_WEST -location 1580.0 {u_vzz_19}
+place_pad -master sky130_ef_io__vddio_hvc_pad -row IO_WEST -location 1500.0 {u_v18_19}
+place_pad -row IO_WEST -location 1420.0 {u_rocc_ctrl_o_s_.u_io}
+place_pad -row IO_WEST -location 1340.0 {u_rocc_ctrl_o_exception_.u_io}
+place_pad -row IO_WEST -location 1260.0 {u_rocc_ctrl_o_host_id_.u_io}
+place_pad -master sky130_ef_io__vccd_hvc_pad -row IO_WEST -location 780.0 {u_vdd_8}
+place_pad -master sky130_ef_io__vssd_hvc_pad -row IO_WEST -location 700.0 {u_vss_8}
+place_pad -master sky130_ef_io__vssio_hvc_pad -row IO_WEST -location 620.0 {u_vzz_20}
+place_pad -master sky130_ef_io__vddio_hvc_pad -row IO_WEST -location 540.0 {u_v18_20}
+place_pad -row IO_WEST -location 220.0 {u_clk_i.u_in}
+
+# Place corners
+place_corners sky130_ef_io__corner_pad
+
+# Place IO fill
+set iofill {sky130_ef_io__com_bus_slice_20um
+            sky130_ef_io__com_bus_slice_10um
+            sky130_ef_io__com_bus_slice_5um
+            sky130_ef_io__com_bus_slice_1um}
+place_io_fill -row IO_NORTH {*}$iofill
+place_io_fill -row IO_SOUTH {*}$iofill
+place_io_fill -row IO_WEST {*}$iofill
+place_io_fill -row IO_EAST {*}$iofill
+
+# Connect ring signals
+connect_by_abutment
+
+# Add bond pads
+place_bondpad -bond sky130_ef_io__bare_pad padring.*_pads\[0\].i0.*.gpio -offset "12.5 115"
+place_bondpad -bond sky130_ef_io__bare_pad padring.*_pads\[0\].i0.*.io* -offset "8 95"
```

Comment:
you need to call `remove_io_rows` at the end to ensure the rows get removed from the DEF

Reply:
done (sync from private started)